Analysis/DEEP2_R23_O32/Binning/N_Bins_Grid_Analysis.py

```diff
@@ -6,8 +6,13 @@
 from astropy.table import Table
 
 
-def n_times_binned(fitspath, dataset, pdf_pages, outfile, n_split, R23, O32, SNR3, data3, galinbin):
+def n_times_binned(fitspath, pdf_pages, outfile, n_split, individual_ID, R23, O32,
+                   SNR3, data3, galinbin):
     """
+    R23 and O32 are going to be log values
+    One_dimensional binning for R23 followed by each bin being split in O32 in n_split bins
+    increase the number of galaxies as R23 increases
+
```

Comment:
The docstrings should provide more information of this function's purpose.

Also, please describe the different parameters.  PyCharm default style is something like this:
```
:param fitspath: str
         Absolute path for working direcotry
```

Reply:
fixed